nmdc_runtime/site/graphs.py

```diff
@@ -467,3 +470,16 @@ def nmdc_study_to_ncbi_submission_export():
         all_instruments,
     )
     ncbi_submission_xml_asset(xml_data)
+
+
+@graph
+def fill_missing_data_generation_data_object_records():
```

Comment:
To me "fill" implies that it's going to actually commit the results to Mongo. Is there any reason this Graph doesn't do that?

Reply:
Hm, so we want to see the JSON export that this job produces, visually QC it and then ingest it into the system. So it's not doing the "filling" automatically. Is "stitch_" a good prefix?